domains/transfer/Transfer.API/API.IntegrationTests/Transfer/Api/Controllers/TransferAgreementsControllerTest.cs

```diff
@@ -47,8 +47,7 @@ public async Task Create_ShouldCreateTransferAgreement_WhenModelIsValid()
         var request = new CreateTransferAgreementProposal(DateTimeOffset.UtcNow.AddMinutes(1).ToUnixTimeSeconds(), null, receiverTin);
         var createdProposalId = await CreateTransferAgreementProposal(request);
 
-        var poWalletServiceMock = SetupPoWalletServiceMock();
-        var receiverClient = factory.CreateAuthenticatedClient(poWalletServiceMock, sub: Guid.NewGuid().ToString(), tin: receiverTin);
+        var receiverClient = factory.CreateAuthenticatedClient(sub: Guid.NewGuid().ToString(), tin: receiverTin);
```

Comment:
How do I now setup a per test PoWalletServiceMock?

Reply:
Get the wallet mock from the factory class and setup mocked calls. But do any tests actually do this now?

Reply:
Don't know, doesn't look like it